exporter/mezmoexporter/README.md

```diff
@@ -0,0 +1,30 @@
+# Mezmo Exporter
+
+This exporter supports sending OpenTelemetry log data to [Mezmo](https://mezmo.com).
+
+# Configuration options:
+
+- `endpoint` (required): LogService's [Region](https://cloud.tencent.com/document/product/614/56473).
+- `ingestion_key` (required): LogService's LogSet ID.
```

Comment:
Should this link to to https://docs.logdna.com/docs/ingestion-key?

Reply:
Yes, corrected.